### PR TITLE
replace Admiral regex

### PR DESCRIPTION
**Pull request by Yuki2718**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [x] Filters maintenance.

## What issue is being fixed?

e.g. https://github.com/AdguardTeam/AdguardFilters/issues/52434

### Add your comment and screenshots

1. Your comment

I'm surprised to see such a too rough rules and the fact they survived with only a handful of exception. But they do not catch most of Admiral domains, say, 
`https://aliasanvil.com/v2buofj3GtkNgN5i_HpKdhXQcR0H-0FUXZz8Ve2S2SeH7GuEEL3YrWAc` from https://github.com/uBlockOrigin/uAssets/issues/16731 - not caught
`https://shiveringspot.com/v2cuhVlPdAl7d8Q7X7QJfIeC77AsxPA02VeJsMnr-GByZturvdBnwStA` from https://github.com/AdguardTeam/AdguardFilters/issues/52434 - not caught
`https://magicaljoin.com/v2csbc1vWhMePZG5y3Rz9Na9buGRwkT-Ct8xF7JTmTUuzhJAeZGsO8G4` - from https://github.com/AdguardTeam/AdguardFilters/issues/142024 - not caught

I've been using the suggested rule here and this actually catches all the above, and I haven't experienced single false positive. I also checked some of those excluded domain and don't see my rule incorrectly blocks something. Only downside is it uses lookahead not compatible with some platforms, but AFAIK the current rules are not compatible with CB too.

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

**Comment by Yuki2718** [comment]
BTW shouldn't the same rule be added to TPF?

File: AnnoyancesFilter/Popups/sections/antiadblock.txt [hunk 1]
```diff
@@ -350,10 +350,7 @@ fosspost.org,portalvirtualreality.ru,pingvin.pro,bonobono.com,curbsideclassic.co
...
-/^https:\/\/.*?\/v2\/0\/[a-zA-Z0-9_-]{30,}$/$third-party,script
-/^https?:\/\/[^\/]+\.[^\/]+\/[a-zA-Z0-9_-]{32,}$/$script,third-party,domain=pwinsider.com|wepc.com|howtogeek.com|boredpanda.com|digitaltrends.com|geekspin.co|cleveland.com|veliainn.com|inquirer.net|winhelponline.com|bloody-disgusting.com|bigleaguepolitics.com|sportskeeda.com|golf.com
+/^https:\/\/[0-9a-z]{7,25}\.com\/v2(?:\/0\/)?(?=[-_0-9a-z]{0,84}[A-Z])(?=[-_a-zA-Z]{0,84}[0-9])[-_0-9a-zA-Z]{54,85}(#\?v=[0-9a-f]{32})?$/$ping,script,xmlhttprequest,third-party,match-case
```
[the remark this answers is not in the document]
Comment (Yuki2718) [reply AnnoyancesFilter/Popups/sections/antiadblock.txt:353]: Sure.

**Comment by Yuki2718** [comment]
Should hyphen and underscore be the last e.g. `[-_0-9a-zA-Z]` => `[0-9a-zA-Z_-]`?

**Comment by zloyden** [comment]
This is not necessary, but it would be nice.

**Review by zloyden — APPROVED** [review]

**Pull request merged**